### DISCUSSION TITLE
Ampliar el diagrama de las states de una SM

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#226] [3. Ideas & Feature Requests] [answered]
Hola

Quisiera ver la posibilidad de ampliar la zona de la vista de una SM, ya que al usar el ctrl + scroll aumenta la vista del navegador.

Dejo el SS de la zona que digo.

Gracias :>

![image](https://user-images.githubusercontent.com/61419471/120523334-c4933780-c3a3-11eb-8cb6-e36cbaeef1e3.png)

**Comment by gmoplaza** [comment, marked as answer]
@NicolasPLG que navegador usas? Ya no debería estar el problema del scroll

**Reply by NicolasPLG** [reply]
hola @gmoplaza toda la razón era el navegador el problema.
Sin embargo me di cuenta que está un poco desalineado en los nombres de la state y tambien veo que el mapa es un poco pequeño para navegar

Dejo aqui el SS.

![image](https://user-images.githubusercontent.com/61419471/120523956-86e2de80-c3a4-11eb-8b66-7cf5732137a1.png)

**Reply by gmoplaza** [reply]
@NicolasPLG estamos concientes de los problemas visuales con los nombres. 
Si estos te causan problemas en el trabajo puedes levantar una solicitud interna y pedir priorizarlo con tu representante Cotalker, lo mismo con el tamaño del mini mapa.
De todas formas creo que la pregunta original ya ha sido resuelta.

**Reply by NicolasPLG** [reply]
si efectivamente cerrare el tema. gracias @gmoplaza
